### PR TITLE
LIVE-997: immersive basic styles

### PR DESCRIPTION
**Pull request by joecowton1**
## Why are you doing this?

Updates the majority of styles for the immersive template. Have added a list of outstanding tasks below screenshots.

## Changes

- Add `Immersive` to to list of templates with no wrapper styles
- Add white text option to Byline Italics
- Add `Immersive` to list of fullwidth image templates
- Add  standfirst styling
- Add  headline styling
- Add series styling
- Add background colour

## Screenshots

| Before | After |
| --- | --- |
|  |  |

## Outstanding tasks for Immersive template:
- Lines (triple-line) component needs padding
- Headline needs pillar colours for white bg variant
- Standfirst needs pillar colours for white bg variant
- Byline needs pillar colours for white bg variant
- Image needs positioning lower on page
- Kicker position needs to move, possibly into headline comopnent

### CODE REVIEW

File: src/components/editions/article.tsx [hunk 1]
```diff
@@ -87,12 +87,16 @@ const galleryWrapperStyles = css`
 	}
 `;
 
-const headerBackgroundStyles = (item: Format): SerializedStyles => css`
+const headerBackgroundStyles = (item: Item): SerializedStyles => css`
```
Comment (faresite) [inline src/components/editions/article.tsx:90]: Don't think you need this change?

File: src/components/editions/byline.tsx [hunk 2]
```diff
@@ -195,8 +219,13 @@ const Byline: FC<Props> = ({ item, shareIcon, large, avatar }) => {
...
 	const { kicker: kickerColor } = getThemeStyles(format.theme);
 
-	const bylineColor =
-		format.design === Design.Media ? neutral[100] : kickerColor;
+	const ignoreKickerColour = (format: Format): boolean => {
+		return (
+			format.design === Design.Media ||
+			format.display === Display.Immersive
+		);
+	};
```
Comment (faresite) [inline src/components/editions/byline.tsx:227]: Nit 
```suggestion
	const ignoreKickerColour = (format: Format): boolean =>
		format.design === Design.Media || format.display === Display.Immersive;
```

File: src/components/editions/headerImage.tsx [hunk 3]
```diff
@@ -62,27 +62,31 @@ const fullWidthCaptionStyles = css`
...
+
 const getStyles = (format: Format): SerializedStyles => {
-	switch (format.design) {
-		case Design.Interview:
-		case Design.Media:
-			return fullWidthStyles;
-		default:
-			return styles;
-	}
+	return isFullWidthImage(format) ? fullWidthStyles : styles;
```
Comment (faresite) [inline src/components/editions/headerImage.tsx:74]: Nice 👍

File: src/components/editions/series.tsx [hunk 4]
```diff
@@ -58,7 +56,11 @@ interface Props {
 const getStyles = (item: Item): SerializedStyles => {
 	const format = getFormat(item);
 	const { kicker } = getThemeStyles(format.theme);
-	if (item.design === Design.Interview) {
+	if (
+		item.design === Design.Interview ||
+		item.display === Display.Immersive ||
+		item.design === Design.Media
```
Comment (faresite) [inline src/components/editions/series.tsx:62]: is this needed for `Design.Media` ?

File: src/components/editions/header.tsx [hunk 5]
```diff
@@ -77,16 +77,24 @@ const galleryHeaderBorderStyles = css`
 	}
 `;
 
-const interviewHeaderStyles = (item: Format): SerializedStyles => css`
-	${from.tablet} {
-		padding-left: ${tabletArticleMargin}px;
-	}
+const interviewHeaderStyles = (item: Item): SerializedStyles => {
```
Comment (abeddow91) [inline src/components/editions/header.tsx:80]: Should the name be updated here now it's not being used exclusively for interview?

File: src/components/editions/headerImage.tsx (hunk 3 above)
Comment (abeddow91) [reply src/components/editions/headerImage.tsx:74]: +1

File: src/components/editions/article.tsx (hunk 1 above)
Comment (joecowton1) [reply src/components/editions/article.tsx:90]: If you check the `headerBackgroundStyles` function it expects an `Item` not a `Format` so Typescript was throwing an error here:

```
export const headerBackgroundColour = (item: Item): Colour => {
	const format = getFormat(item);
	[...]
}
```

File: src/components/editions/byline.tsx (hunk 2 above)
Comment (joecowton1) [reply src/components/editions/byline.tsx]: Fixed.

File: src/components/editions/series.tsx (hunk 4 above)
Comment (joecowton1) [reply src/components/editions/series.tsx]: Good shout 👍

File: src/components/editions/header.tsx (hunk 5 above)
Comment (joecowton1) [reply src/components/editions/header.tsx]: Good idea!

**Review by abeddow91 — APPROVED** [review]
Nice one!

File: src/components/editions/article.tsx (hunk 1 above)
Comment (faresite) [reply src/components/editions/article.tsx:90]: @joecowton1 maybe I'm missing something, but `headerBackgroundColour` doesn't need `Item` either? in fact it needs format, hence uses `getFormat`?

Comment (joecowton1) [reply src/components/editions/article.tsx]: Updated 👍

**Review by faresite — APPROVED** [review]
Awesome 👍

**Pull request merged**